docs/06-concepts/02-models.md

```diff
@@ -86,7 +86,7 @@ values:
 
 :::info
 
-It's recommended to always set `serialized` to `byName` in any new Enum models, as this is less fragile and will be changed to the default setting in version 2 of Serverpod.
+It's recommended to always set `serialized` to `byName` in any new Enum models, as this is less fragile and will be changed to the default setting in version 3 of Serverpod.
```

Comment:
Suggestion: Could you add this change to the already deployed version of the docs as well? You will find it under the `versioned_docs` directory.